data.ts

```diff
@@ -1,3 +1,21 @@
+export interface CashBalanceReport {
+  companyId: string
+  id: string
+  assets: ReportSection
+  liabilitiesAndEquity: ReportSection
+}
+
+export interface ReportSection {
+  nestedSection?: ReportSection[]
+  accounts?: Account[]
+}
```

Comment:
```suggestion
export interface ReportSection {
  nestedSection: ReportSection[] | []
  accounts: Account[]
}
```
thankfully, this is not an endlessly recursive report